lib/opentracing/span.rb

```diff
@@ -40,11 +40,20 @@ def get_baggage_item(key)
       nil
     end
 
+    # ** Deprecated **
     # Add a log entry to this span
     # @param event [String] event name for the log
     # @param timestamp [Time] time of the log
     # @param fields [Hash] Additional information to log
     def log(event: nil, timestamp: Time.now, **fields)
```

Comment:
Why don't we remove `event: nil` and not create a new log_kv method? Are we planning to use `log` for something else? I think this should also be explained in the commit message.

Reply:
@indrekj Thanks for the feedback.  Added reasons for the method name replacement in the code and commit message.